backend/src/activitypub/objects/index.ts

```diff
@@ -129,16 +130,19 @@ export async function cacheObject(
 
 	{
 		const properties = JSON.parse(row.properties)
+
+		// prettier-ignore
 		const object = {
 			published: new Date(row.cdate).toISOString(),
-			...properties,
 
 			type: row.type,
 			id: new URL(row.id),
 
```

Comment:
It's unnecessary to set `type: row.type` and `id: new URL(row.id),` because `row.properties === sanitizedProperties`. So, the expansion `...sanitizedProperties` should suffice